torch/_export/converter.py

```diff
@@ -224,6 +224,11 @@ def get_dtype_as_int(tensor):
     return _TORCH_DTYPE_TO_ENUM[dtype]
 
 
+def as_tensor(data, dtype, device):
+    dtype = _TORCH_ENUM_TO_DTYPE[dtype] if isinstance(dtype, int) else dtype
+    return torch.tensor(data, dtype=dtype, device=device)
```

Comment:
aten::as_tensor has many variants. 

This works for aten::as_tensor.list. 
what about 
 "aten::as_tensor(Tensor data, *, ScalarType? dtype, Device? device) -> Tensor"
 aten::as_tensor.float
 aten::as_tensor.int